sdk/python/tests/integration/offline_store/test_universal_historical_retrieval.py

```diff
@@ -370,6 +370,9 @@ def test_historical_features(environment, universal_data_sources, full_feature_n
         full_feature_names=full_feature_names,
     )
 
+    if job_from_df.supports_remote_storage_export:
```

Comment:
should we be doing a more thorough test of this behavior? e.g. after unloading, perhaps retrieving and combining to ensure that the data is the same?

not sure how much info we get from these two lines

Reply:
I'm gonna do that in a separate PR to unblock release